README.md

```diff
@@ -222,6 +221,6 @@ I would like to thank everyone from whom inspiration was obtained toward the dev
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT) licensed.

```

Comment:
- [ ] This link is not appropriate! Please accept my suggestion!

Correct link: [MIT.md](https://github.com/amon-cofie/blog-app/blob/feature/build_models/MIT.md)


``` suggestion
This project is [MIT](./MIT) licensed.
This project is [MIT](./MIT.md) licensed.
```